modules/ROOT/pages/api-groups-manage-api-group-instance-contracts.adoc

```diff
@@ -0,0 +1,80 @@
+= Manage API Group Instance Contracts
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+After you grant access to API Group instances in Exchange, a contract is automatically created between an application and the API instance. You can then manage that contract from API Manager.
+
+== Applied Contracts
+
+With API Groups, an application can have multiple contracts with the same API in these scenarios:
+*  An application requests access to an API, and the request is approved. In this case, a direct contract is created between the application and the API. This is known as _API instance contracts_.
+* An application requests access to an API Group instance, which has an API instance with which the application already has a direct contract. This is known as _API Group instance contracts_.
+
+When an application has multiple contracts with the same API instance, only one contract is active at any given time. This is known as _Applied contracts_.
+
+.The screen capture displays the contract levels and the contract status for the API instance contracts.
+image:api-groups-instance-contracts.png[API Group Instance Contracts,95%,100%]
+
```

Comment:
These callouts are spaced but the previous ones are not?

Reply:
Done! Thanks!